pkg/providers/aws/config.go

```diff
@@ -79,6 +80,38 @@ func NewDefaultConfigMapConfigManager(client client.Client) *ConfigMapConfigMana
 	return NewConfigMapConfigManager(DefaultConfigMapName, DefaultConfigMapNamespace, client)
 }
 
+func (m *ConfigMapConfigManager) ReconcileNetworkStrategy(ctx context.Context, c client.Client, cidr string) error {
+	cm, err := resources.GetConfigMapOrDefault(ctx, m.client, types.NamespacedName{Name: m.configMapName, Namespace: m.configMapNamespace}, BuildDefaultConfigMap(m.configMapName, m.configMapNamespace))
+	if err != nil {
+		return errorUtil.Wrapf(err, "failed to get aws strategy config map %s in namespace %s", m.configMapName, m.configMapNamespace)
+	}
+	var network struct {
+		Production struct {
```

Comment:
Need to improve this so it works for all tiers.

Reply:
do we need this? We can use the configManager.ReadStorageStrategy(ctx, providers.NetworkResourceType, tier) to get the the data from the config map

Reply:
Resolving this one as I've removed the logic to update the config map as discussed 👍